src/resources/js/pages/sessions/test-steps/index.vue

```diff
@@ -291,5 +297,18 @@ export default {
             required: true,
         },
     },
+    methods: {
+        jsonToCurl: function (request) {
+            let result = 'curl --location --request ';
+
+            result = result.concat(request['method']," 'Session_URL",request['uri'],"' \\  ")
+            for (let h in request['headers']) {
+                result += "--header '"+h+": "+ request['headers'][h][0]+"' \\  "
```

Comment:
You can use template strings to make this slightly tidier:
```suggestion
                result += `--header '${h}: ${request['headers'][h][0]}' \\  `;
```

You may in fact want to restructure the function to make heavier use of template strings, something like this:

```js
let headers = "";
for (h in headers) {
   headers += `--header ${...}`
}
const body = `--data-raw ${JSON.stringify(...)}`;
const result = `curl --location --request ${headers} ${body}`
```